src/jshint.js

```diff
@@ -4225,7 +4225,7 @@ var JSHINT = (function() {
     var i = 0;
     var inof = ["in", "of"];
     var level = 0; // BindingPattern "level" --- level 0 === no BindingPattern
-    var comma; // First comma punctuator at level 0
+    var firstcomma; // First comma punctuator at level 0
```

Comment:
That may be, but I don't think that's the right approach to fixing it --- the reason is that the old names were more descriptive and easier to understand.

A better approach would be installing all of the global functions as properties on an object, so that they can be fully qualified when needed